graphene_plugin.py

```diff
@@ -193,15 +222,15 @@ def create_attribute_type(ctx: ClassDefContext, attribute_node: AssignmentStmt)
         return None
 
     argument_node_names = attribute_node.rvalue.arg_names  # type: ignore[attr-defined]
-    return_type = get_python_type_from_graphene_type(ctx, argument_nodes)
+    return_type = get_python_type_from_graphene_type(ctx, argument_nodes, argument_node_names, covariant=True)
 
     argument_list: List['ArgumentNode'] = []
     names_to_nodes = zip(argument_node_names[1:], argument_nodes[1:])
 
     for argument_name, argument_node in names_to_nodes:
         if argument_name in ['description', 'required', 'default_value', 'deprecation_reason']:
             continue
-        python_type = get_python_type_from_graphene_type(ctx, [argument_node], argument_node_names)
+        python_type = get_python_type_from_graphene_type(ctx, [argument_node], argument_node_names, covariant=False)
```

Comment:
So `Sequence` won't show up as argument types, but will for return types. I guess the value is that you're less likely to have to annotate a variable that's going to get returned, makes sense.

Reply:
This is really just because `List[]` is checked invariantly but we want covariant checking (which is what `Sequence` provides.)

More concretely, if a field definition says that the resolver's return type _should_ be `List[Union[HumanModel, PersonModel]]` but the _actual_ resolver's return type is `List[HumanModel]`, and error **would** be raised because `List[]` is checked invariantly.

However if the resolver's return type _should_ be `Sequence[Union[HumanModel, PersonModel]]` and the actual return type is `List[HumanModel]` then that **passes** (which is desirable) because that is covariantly sound.

But for arguments we just wanna check that the function is typed correctly using the _exact_ correct types. So we don't want to use the covariant `Sequence`, we really do want `List`.